modules/compose.mjs

```diff
@@ -192,7 +195,15 @@ class ReplyWithHeader {
         if (this.isForward) {
             let mozForwardContainer = this._getByClassName('moz-forward-container');
             this._cleanNodesUpToClassName(mozForwardContainer, targetNodeClassName);
-            mozForwardContainer.insertAdjacentElement(positionAfterBegin, this._createElement('br'));
+			
+			// Insert 2 <br> before the headers to make it look like a reply does.
+			mozForwardContainer.insertAdjacentElement(positionAfterBegin, this._createElement('br'));
+			mozForwardContainer.insertAdjacentElement(positionAfterBegin, this._createElement('br'));
+
+			// There are 2 <br> originally, but none for reply mode.
+			// We add one for replies, so let's remove one for forwards to make it even.
+			let nextBr = mozForwardContainer.querySelector("div.moz-email-headers-table + br");
+			nextBr?.remove();
```

Comment:
@ygoe Can you please format this code block correctly?

I'm okay with adding a new `br,` but let's remove these lines from 203 to 206. TB does not always play well.

Reply:
What's to format here? Looks good to me. Please clarify.

I've changed the brs so that it matches exactly, line by line, the behaviour of Thunderbird without this add-on, for both reply and forward. So I was actually minimising the distortions this add-on introduces.

What does "play well" mean here? What would it do when what was expected? I don't have any information on that, and I've successfully tested this in the only supported version of Thunderbird.

Reply:
Maybe without a signature it's different? I have one. And I admit I haven't tested without a signature.

Reply:
@ygoe I'm sorry, but I forgot to mention the code format. Your new code lines have an additional tab in them; kindly align it with the `if` block code alignment.

Please remove these lines from 203 to 206.